### DISCUSSION TITLE
Can't delete last item in vault as it overflows the screen

### DISCUSSION THREAD

**Discussion by keval6b** [dani-garcia/vaultwarden#3204] [Q&A] [answered]
### Subject of the issue
Can't delete last item in vault as it overflows the bottom of the screen

### Deployment environment

* Vaultwarden version: v1.27.0
* Web-vault version: v2022.12.0
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.39.2
* Clients used: 
* Reverse proxy and version: 

### Steps to reproduce

Add a lot of items to the vault

### Expected behaviour

Able to see and press the delete button for the last item

### Actual behaviour

Can't see or press the delete button for the last item

### Troubleshooting data

![vaultwarden-ui-issue](https://user-images.githubusercontent.com/33595996/215541063-78308e16-37d0-4d01-91b7-39396744cbad.png)

**Comment by BlackDex** [comment, marked as answer]
This is a client GUI issue it looks like. Those sound be reported to Bitwarden.

As a workaround i suggest to search for this item which makes probably a more top item after which you should be able to delete this item.

**Reply by keval6b** [reply]
Ah my bad, thanks. Opened: https://github.com/bitwarden/clients/issues/4599
